sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md

```diff
@@ -42,10 +42,9 @@ appropriate SAS token with write access and connection string. To make this poss
 
 ### Checkpointing
 
-Checkpointing is a process by which readers mark or commit their position within a partition event sequence.
-Checkpointing is the responsibility of the consumer and occurs on a per-partition basis within a consumer group.
-This responsibility means that for each consumer group, each partition reader must keep track of its current position
-in the event stream, and can inform the service when it considers the data stream complete. If a reader disconnects from
+Checkpointing is a process during which consumers mark their reading position in a partition of event stream.
+Checkpointing occurs on a per-partition basis within a consumer group meaning that within each consumer group, reader for each partition must keep track of its own current reading position
+in the event stream, and can inform the service when its reading cursor reaches the stream end. If a reader disconnects from
```

Comment:
The consumer determines what the checkpoint should be on the stream right? The previous explanation seems to be a better fit